### PR TITLE
appium-server 0.0.4

### PR DESCRIPTION
**Pull request by moatazeldebsy**
### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/appium-server/0.0.4/step.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  commit: 6a427500aee5a429f61d1bda9632a7e30ec455d0
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- android
+- react-native
+type_tags:
+- utility
```
Comment (ofalvai) [inline steps/appium-server/0.0.4/step.yml:21]:
```suggestion
- test
```

I think test would be a better category for this step

File: steps/appium-server/0.0.4/step.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  brew:
+  - name: nodejs
+  apt_get:
+  - name: nodejs
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- appium_version: v1.21.0
```
Comment (ofalvai) [inline steps/appium-server/0.0.4/step.yml:35]: Are you sure it's working with the `v` prefix?

**Review by ofalvai — CHANGES_REQUESTED** [review]
Just a few minor comments, otherwise looks good

File: steps/appium-server/0.0.4/step.yml (hunk 2 above)
Comment (moatazeldebsy) [reply steps/appium-server/0.0.4/step.yml:35]: yes, you can check the release here [https://github.com/appium/appium/releases](https://github.com/appium/appium/releases)

File: steps/appium-server/0.0.4/step.yml (hunk 1 above)
Comment (moatazeldebsy) [reply steps/appium-server/0.0.4/step.yml:21]: I think because I'm installing the server only so I added it as `utility`

**Review by ofalvai — APPROVED** [review]

**Pull request merged**